lib/lib.misc.php

```diff
@@ -998,7 +998,12 @@ function rejudging_finish($rejudgingid, $request, $userid = NULL, $show_progress
 	auditlog('rejudging', $rejudgingid, $request.'ing rejudge', '(start)');
 
 	while ( $row = $res->next() ) {
-		if ( $show_progress ) echo "s" . specialchars($row['submitid']) . ", ";
+		if ( $show_progress ) {
+			echo "s" . specialchars($row['submitid']) . ", ";
+			// Also flush the output, so the browser displays updates
+			ob_implicit_flush(true);
+			ob_flush();
```

Comment:
Do we need this at all? See the first user contributed example here: http://php.net/manual/en/function.ob-implicit-flush.php

Reply:
Let me test this :)